tol_lab_share/error_codes.py

```diff
@@ -75,7 +75,7 @@ def origin_for_json(self) -> Any:
             return "sample"
         return "root"
 
-    def json(self) -> dict[str, Any]:
+    def json(self) -> "dict[str, Any]":
```

Comment:
Out of interest, why was this necessary?  I know Python supports string descriptions of types, but it usually only needs this when you have a circular reference.

Reply:
I don't really think the quotes are required because we're working with Python 3.11 (which is > 3.9) but Pylance suggested me this.

![image](https://github.com/user-attachments/assets/d287da40-55b0-4e32-9497-91f0c0269a22)

Reply:
Hmmm.  Even if this were true, the correct fix is to use `Dict` instead of `dict` and import it from `typing` but that shouldn't be necessary as you said.